packages/ng-primitives/date-picker/src/date-picker-date-button/date-picker-date-button.ts

```diff
@@ -136,8 +136,8 @@ export class NgpDatePickerDateButton<T> implements OnDestroy {
   @HostListener('keydown.enter', ['$event'])
   @HostListener('keydown.space', ['$event'])
   protected select(event?: KeyboardEvent): void {
-    // if the button is disabled, or is already selected, do nothing.
-    if (this.disabled() || this.selected()) {
```

Comment:
Not quite sure about the implications of removing this. Maybe you got some more insights if it might lead to some problems down the line for the regular date picker

Reply:
I actually don't recall specifically, but I have done some testing and everything seems to work fine, so hopefully it is ok 😊